aries-site/src/examples/templates/global-header/HeaderNavItems.js

```diff
@@ -0,0 +1,51 @@
+import React, { useEffect, useState } from 'react';
+import { Box, CheckBox, Button, Text } from 'grommet';
+import { HeaderNav, UserContext } from '.';
+
+const defaultUser = {
+  firstName: 'John',
+  lastName: 'Doe',
+  email: 'john.doe@hpe.com',
+  image: '//s.gravatar.com/avatar/99020cae7ff399a4fbea19c0634f77c3?s=80',
+};
+
+export const HeaderNavItems = () => {
+  const [user, setUser] = useState(defaultUser);
+  const [checked, setChecked] = useState(true);
```

Comment:
Readability... soft opinion
```suggestion
  const [showProfile, setShowProfile] = useState(true);
```

Reply:
Done.